### PR TITLE
v0.7.0 Release Candidate Preparation

### PR DESCRIPTION
**Pull request by safl**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -33,8 +40,9 @@ The following people have contributed to the implementation of xNVMe.
 * Krishna Kanth Reddy <krish.reddy@samsung.com>
 * Mads Ynddal <m.ynddal@samsung.com>
 * Michael Bang <mi.bang@samsung.com>
+* Niclas Hedam <niclas@hed.am> 
 * Pierre Labat <plabat@micron.com>
-* Rishabh Shukla <rishabh.sh@samsung.com>
+* Rishabh Shukla <rishabh.sh@samsung.com> 
 * Simon A. F. Lund <os@safl.dk>
 * Simon A. F. Lund <simon.lund@samsung.com>
 * Vered Kalef <vered.kalef@gmail.com>
```
Comment (ankit-sam) [inline CONTRIBUTORS.md:48]: Vikash Kumar should be added to previous release contributors

**Review by ankit-sam — COMMENTED** [review]
Everything else looks good. Just one comment

File: man/meson.build [hunk 2]
```diff
@@ -1,42 +1,3 @@
-# SPDX-FileCopyrightText: Samsung Electronics Co., Ltd
```
Comment (karlowich) [inline man/meson.build:1]: Is removing the Copyright intentional?

Comment (karlowich) [reply man/meson.build:1]: Also is removing the manpages for all of these tools intentional, looks a little off...

File: docs/autogen/apigen.py [hunk 3]
```diff
@@ -156,19 +156,25 @@ def main(args):
...
+        "xnvme_znd",
+        "xnvme_lba",
+        "xnvme_queue",
+        "xnvme_libconf",
         "xnvme_opts",
         "xnvme_spec",
         "xnvme_util",
         "xnvme_ver",
-        "xnvme_znd",
-        "xnvmec",
```
Comment (karlowich) [inline docs/autogen/apigen.py:171]: What's going on here? 
Maybe they should be alphabetical?

File: CHANGELOG.rst [hunk 4]
```diff
@@ -18,6 +18,52 @@ Known Issues
...
+
+* API / Command-Sets
+  * Support for FDP and DSM
+  * Support for Key-Value SSDs
+
+* Backends
+  * The POSIX backend is replaced by CBI -- Common Backend Implementation
+  * Instead of "POSIX" then a handful of commonly useful implementations
+    are provided, being "strictly" POSIX was an impractical restriction,
+    such as lagging support for features which are provided on modern
```
Comment (karlowich) [inline CHANGELOG.rst:47]: lagging -> lacking

**Review by karlowich — CHANGES_REQUESTED** [review]
Overall looks good!
A few comments and questions

File: docs/autogen/apigen.py (hunk 3 above)
Comment (safl) [reply docs/autogen/apigen.py:171]: Yeah, sorted now. As I recall, then there used to be a dependency on the ordering of these for the api-generator.

File: CHANGELOG.rst (hunk 4 above)
Comment (safl) [reply CHANGELOG.rst:47]: Thanks, fixed!

File: man/meson.build (hunk 2 above)
Comment (safl) [reply man/meson.build:1]: > Also is removing the manpages for all of these tools intentional, looks a little off...

They are not removed? They are updated and then there are additions for e.g. ``kvs``.

**Review by ankit-sam — APPROVED** [review]

File: man/meson.build [hunk 5]
```diff
@@ -1,42 +1,3 @@
...
-install_man('xnvme-info.1')
-install_man('xnvme-library-info.1')
-install_man('xnvme-list.1')
-install_man('xnvme-log-erri.1')
-install_man('xnvme-log-health.1')
-install_man('xnvme-log.1')
-install_man('xnvme-padc.1')
-install_man('xnvme-pioc.1')
-install_man('xnvme-sanitize.1')
-install_man('xnvme.1')
```
[the remark this answers is not in the document]
Comment (safl) [reply man/meson.build:39]: > It looks to me like all of these manpages will no longer be installed. Am I mistaken?

Ohh, now I see what you mean... hmm... let me check up on that. Thanks for catching it!

[the remark this answers is not in the document]
Comment (karlowich) [reply man/meson.build]: Great!

**Pull request merged**